src/Core/Helpers/AbstractConfigHelper.php

```diff
@@ -33,16 +33,22 @@ abstract class AbstractConfigHelper {
   const INVENTORY_ITEMS_PER_PAGE = 500;
 
   /**
+   * Retrieve the configuration client id for the global client
+   *
    * @return mixed
    */
   abstract public function getClientId();
 
   /**
+   * Retrieve the configuration client secret for the global client
    * @return mixed
    */
   abstract public function getClientSecret();
 
   /**
+   * Checks if an item already exists in cache and returns the key,
+   * if the item does not exists in cache adds item to cache
```

Comment:
- the cache etc are implementation details that should not be mentioned in the contract.
- what does "item" mean? this looks like it is getting a very specific value, called an Order Referrer Key.